### PR TITLE
Daniel web branch

### PR DESCRIPTION
**Pull request by danielzar2710**
Mostly design and usability changes , little server side changes

### CODE REVIEW

**Review by NajibAbuSamra — COMMENTED** [review]
why so many files added to the classpath? 😮

**Comment by danielzar2710** [comment]
beats me

**Comment by danielzar2710** [comment]
it works on my computer

**Comment by danielzar2710** [comment]
and all the paths are relative

File: src/mm/webclientservlets/GetAllPairs.java [hunk 1]
```diff
@@ -45,13 +45,15 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
 		System.out.println("Get AllPairs Servlet");
 
 		ArrayList<Pair> pairsArray = new ArrayList<Pair>();
-		ArrayList<PairsInfo> pairsMainInfo = new ArrayList<PairsInfo>();
```
Comment (NajibAbuSamra) [inline src/mm/webclientservlets/GetAllPairs.java:48]: @firdos94  @danielzar2710  doesn't the PairsInfo model have more info than Pair??

**Review by NajibAbuSamra — COMMENTED** [review]
@danielzar2710  "it works on my computer" - famous last words

File: src/mm/webclientservlets/GetAllPairs.java (hunk 1 above)
Comment (danielzar2710) [reply src/mm/webclientservlets/GetAllPairs.java:48]: no

Comment (danielzar2710) [reply src/mm/webclientservlets/GetAllPairs.java:48]: it's missing some importent data

Comment (danielzar2710) [reply src/mm/webclientservlets/GetAllPairs.java:48]: My version works when i get apairs now

Comment (NajibAbuSamra) [reply src/mm/webclientservlets/GetAllPairs.java:48]: so if pair is enough, do we still need PairsInfo at all?

**Pull request merged**